latest/ug/nodes/hybrid-nodes-cluster-update.adoc

```diff
@@ -129,3 +129,110 @@ aws eks describe-cluster \
 . Choose *Save changes* to finish. Wait for the cluster status to return to *Active*.
 
 . Continue to <<hybrid-nodes-cluster-prep>>.
+
+[#hybrid-nodes-cluster-update-config] 
+== Update hybrid nodes configuration on an existing cluster
+
+You can modify `remoteNetworkConfiguration` on an existing hybrid cluster using:
+
+* <<hybrid-nodes-cluster-update-cfn,{aws} CloudFormation>>
+* <<hybrid-nodes-cluster-update-cli,{aws} CLI>>
+* <<hybrid-nodes-cluster-update-console,{aws-management-console}>>
+
+[#hybrid-nodes-cluster-update-cfn] 
+=== Update hybrid configuration - {aws} CloudFormation
+
+. Update your CloudFormation template with the new network CIDR values.
++
+[source,yaml,subs="verbatim,attributes"]
+----
+RemoteNetworkConfig:
+  RemoteNodeNetworks:
+    - Cidrs: [NEW_REMOTE_NODE_CIDRS]
+  RemotePodNetworks:
+    - Cidrs: [NEW_REMOTE_POD_CIDRS]
+----
+
+. Update your CloudFormation stack with the modified template and wait for the stack update to complete.
+
+[#hybrid-nodes-cluster-update-cli] 
+=== Update hybrid configuration - {aws} CLI
+
+. To modify the remote network CIDRs, run the following command. Replace the values with your desired settings:
++
+[source,bash,subs="verbatim,attributes"]
+----
+aws eks update-cluster
+--name CLUSTER_NAME
+--region AWS_REGION
+--remote-network-config '{"remoteNodeNetworks":[{"cidrs":["NEW_REMOTE_NODE_CIDRS"]}],"remotePodNetworks":[{"cidrs":["NEW_REMOTE_POD_CIDRS"]}]}'
+----
+  * If you don't want to modify `RemoteNodeNetworks` or `RemotePodNetworks`, you can leave that field out from the update request. The existing configuration for these fields will be retained without being overwritten or removed.
+
+
+. Wait for the cluster status to return to ACTIVE before proceeding.
+
+[#hybrid-nodes-cluster-update-console] 
+=== Update hybrid configuration - {aws-management-console}
+
+. Open the Amazon EKS console at link:eks/home#/clusters[Amazon EKS console,type="console"].
+. Choose the name of the cluster to display your cluster information.
+. Choose the *Networking* tab and choose *Manage*.
+. In the dropdown, choose *Remote networks*.
+. Update the CIDRs under `Remote node networks` and `Remote pod networks - Optional` as needed.
+. Choose *Save changes* and wait for the cluster status to return to *Active*.
+
+
+[#hybrid-nodes-cluster-disable]
+== Disable Hybrid nodes on an existing Hybrid cluster
```

Comment:
suggest saying "Disable hybrid nodes on an existing cluster"

for this and the CFN + console examples